aci/utils.go

```diff
@@ -352,3 +352,20 @@ func getOldObjectsNotInNew(keyName string, oldSet, newSet *schema.Set) (oldObjec
 	}
 	return oldObjects
 }
+
+// Compares two list-of-strings and sends a new list-of-strings with only unique strings from the first list.
+// Reversing the two lists generates a new list with different outputs
+func getStringsFromListNotInOtherList(previousValueList interface{}, newValueList interface{}) (generatedList []interface{}) {
```

Comment:
Since this function can be reused it would be good to change its name and behaviour by giving it the ability to generate unique lists from the old and new lists as the input.

```
func getUniqueLists(oldValuesList interface{}, newValuesList interface{}) (UniqueListForOld, UniqueListForNew []interface{}) {
	var difference []interface{}
	for i := 0; i < 2; i++ {
		if i == 1 {
			oldValuesList, newValuesList = newValuesList, oldValuesList
			UniqueListForOld = difference
			difference = nil
		}
		for _, oldValue := range oldValuesList.([]interface{}) {
			found := false
			for _, newValue := range newValuesList.([]interface{}) {
				if oldValue == newValue {
					found = true
					break
				}
			}
			if !found {
				difference = append(difference, oldValue)
			}
		}
	}
	UniqueListForNew = difference
	return UniqueListForOld, UniqueListForNew
}
```